apps/mull-ui/src/utilities.ts

```diff
@@ -96,6 +97,16 @@ export const hasEmoji = (text: string) => {
   return !!emojiRegexRGI().exec(text);
 };
 
+const textLength = string.length;
+export const exceedsNbOfLines = (text: string) => {
+  let countLines = 1;
+  for (let i = 0; i < textLength; i++) if (text.charAt(i) === '\n') countLines++;
+  if (countLines > 6) {
+    return false;
+  }
+  return true;
+};
```

Comment:
⚠So, a few issues

Your code here doesn't prevent me from writing a desc with more than 6 lines.
![image](https://user-images.githubusercontent.com/23544999/113898605-05067880-979a-11eb-992a-8559fa3ae308.png)

This is because of line 87, which for some bizarre reason you took `string.length` as the max number to loop through. This value will always be 0, since you're taking the length of the string type (idk why haha).

I'd recommend a much simpler approach that doesn't need a separate function, using regex. Here's an example
![image](https://user-images.githubusercontent.com/23544999/113899116-79d9b280-979a-11eb-82c1-02e191818fca.png)

Reply:
I hastily opened the PR without double checking the functionality after adding a new line which broke the 6 lines restriction. That's my bad.
Thank you for the regex restriction, but I tried several combination of regex and it unfortunately counts every space instead and not each new line (not to mention how the `match` added an extra console error to fix), so I decided to stick with my for loop instead.

Reply:
> but I tried several combinations of regex and it, unfortunately, counts every space instead and not each new line

Depends on what regex you use. Did you try mine? It works pretty much exactly like your does right now
![image](https://user-images.githubusercontent.com/23544999/113987562-8bfc3500-981c-11eb-9595-ad5f09d85a6f.png)
![image](https://user-images.githubusercontent.com/23544999/113987573-90c0e900-981c-11eb-8cad-ef142c14a0ee.png)

> not to mention how the `match` added an extra console error to fix

Not really, have a look at my suggestion below :)